roles/glance/templates/etc/glance/policy.json

```diff
@@ -16,8 +16,13 @@
     "upload_image": "",
 
     "delete_image_location": "",
+{% if allow_image_locations|default('False')|bool %}
     "get_image_location": "",
     "set_image_location": "",
+{% else %}
+    "get_image_location": "role:admin or role:cloud_admin",
```

Comment:
Have same question as @zfeldstein, seems like permission assignment is reversed i.e when true assign permission to admin and cloud_admin else leave it empty.